submodules/wizard/wizard.js

```diff
@@ -1206,24 +1198,188 @@ define(function(require) {
+					monster.ui.tooltips($template);
+
+					self.wizardReviewBindEvents({
+						template: $template
+					});
 
-			// TODO: Not implemented
+					return $template;
+				};
+
+			self.wizardRenderStep({
+				container: $container,
+				loadData: function(asyncCallback) {
+					self.serviceItemsListingRender({
+						planIds: data.servicePlan.selectedPlanIds,
+						container: $template.find('#service_plan_aggregate'),
+						showProgressPanel: false,
+						success: function() {
+							asyncCallback(null);
+						},
+						error: function(err) {
+							asyncCallback(null);
+						}
+					});
+				},
+				initTemplate: initTemplate
+			});
 		},
 
+		/**
+		 * Utility funcion to extract Review data. Not used, as this is only a review step, so it
+		 * does not provide any new data.
+		 * @param  {jQuery} $template  Step template
+		 * @returns  {Object}  Object that contains a `valid` flag value
+		 */
 		wizardReviewUtil: function($template) {
 			var self = this;
 
-			// TODO: Not implemented
-
 			return {
-				valid: true,
-				data: {}
+				valid: true
 			};
 		},
 
+		/**
+		 * Fomat the wizard data to be rendered for review
+		 * @param  {Object} data  Wizard data
+		 */
+		wizardReviewFormatData: function(data) {
+			var self = this,
+				wizardAppFlags = self.appFlags.wizard,
+				formattedData = _
+					.chain(data)
+					.cloneDeep()	// To not to alter data to save
+					.merge({
+						generalSettings: {
+							accountInfo: {
+								// Set formatted address line 3
+								addressLine3: self.getTemplate({
+									name: '!' + self.i18n.active().accountsApp.wizard.steps.review.generalSettings.formats.addressLine3,
+									data: data.generalSettings.accountInfo
+								})
+							}
+						}
+					})
+					.value(),
+				featureTreeToList = function(features) {
+					return _.flatMap(features, function(feature) {
+						var flattenedFeatures = _.concat([ feature ], featureTreeToList(feature.features));
+						return flattenedFeatures;
+					});
+				};
+
+			// Replace language code with language name
+			formattedData.generalSettings.accountInfo.language = monster.util.tryI18n(monster.apps.core.i18n.active().monsterLanguages, formattedData.generalSettings.accountInfo.language);
+
+			// Set full name for account admins
+			_.each(formattedData.generalSettings.accountAdmins, function(admin) {
+				admin.fullName = monster.util.getUserFullName({
+					first_name: admin.firstName,
+					last_name: admin.lastName
+				});
+				delete admin.firstName;
+				delete admin.lastName;
+			});
+
+			// Replace representative's userId with its full name
+			if (_.has(formattedData.accountContacts, 'salesRep.representative')) {
+				formattedData.accountContacts.salesRep.representative = _
+					.chain(self.wizardGetStore('accountUsers'))	// At this point all the required data has been stored, so we can get it directly
+					.find({
+						id: formattedData.accountContacts.salesRep.representative
+					})
+					.thru(function(user) {
```

Comment:
You can apply the modifier directly here:
```suggestion
					.thru(monster.util.getUserFullName)
```